backend/tests/unit/records/marshmallow/literature/common/test_external_system_identifier.py

```diff
@@ -58,7 +58,7 @@ class TestSchema(Schema):
                 "url_name": "OSTI Information Bridge Server",
             },
             {
-                "url_link": "https://zbmath.org/?q=an%3Azblatt-id",  # noqa
+                "url_link": "http://www.zentralblatt-math.org/zmath/en/search/?an=zblatt-id",  # noqa
```

Comment:
Why this change in this PR?

Reply:
looks like a wrong conflict resolution during rebase. I made the change to `zbmath.org` recently.